packages/core/chrome/core-chrome-browser/src/project_navigation.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Since this file only exports types, maybe it should be organized under `./types/project_navigation.ts` ?

Reply:
If I understand correctly, the `core-chrome-browser` _is_ the package to export all the types.

Then the `core-chrome-browser-internal` is where the implementation lives.